### ISSUE TITLE
Implmenet CSS3 caret-color property

### ISSUE THREAD

**Issue by jabcreations** [Fyrd/caniuse#3081]
The CSS3 caret-color property should be added not simply for the sake of customization though also for accessibility purposes. I'm also dealing with some heavy bugs in Gecko/Firefox including invisible caret in rich text editing and promoting this CSS property would help with UX issues until Mozilla is able to address such issues. On top of that it's been stated as being low-hanging fruit.

https://www.w3.org/TR/css-ui-3/#caret-color
https://www.w3.org/TR/css-ui-4/#caret-color

Implementation is an active topic this month at Mozilla for Gecko:
https://bugzilla.mozilla.org/show_bug.cgi?id=1063162

A request for WebKit/Safari:
https://bugs.webkit.org/show_bug.cgi?id=166572

**Comment by xfq** [comment]
And Blink: https://bugs.chromium.org/p/chromium/issues/detail?id=665422

**Comment by Schweinepriester** [comment]
+1

**Comment by lotap** [comment]
If anyone finds this thread while searching for a compatibility table:
https://developer.mozilla.org/en-US/docs/Web/CSS/caret-color#Browser_compatibility

**Comment by ginlime** [comment]
+1

**Comment by Fyrd** [comment]
Now available at https://caniuse.com/#feat=css-caret-color
